modules/ROOT/pages/administration/index.adoc

```diff
@@ -1,82 +1,21 @@
 = Administration
 
-Cypher contains a number of administrative commands which allows for efficient and sophisticated database management, alias management, server management, and role-based access-control. 
+The pages previously in this chapter have been moved to the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/[Operations Manual].
 
-More information about each of these topics can be found in the following sections:
+More specific information about the content relocation is listed in the table below:
 
-* xref:administration/databases.adoc[]
-* xref:administration/aliases.adoc[]
-* xref:administration/servers.adoc[]
-* xref:administration/access-control/index.adoc[]
-
-[[administration-syntax]]
-== Syntax summaries
-
-Almost all administration commands have variations.
-The most common are parts of the command that are optional or that can have multiple values.
-
-See below a summary of the syntax used to describe all versions of a command.
-These summaries use some special characters to indicate such variations.
-
-.Special characters in syntax summaries
-[options="header", width="100%", cols="1a,3a,3a"]
+[options="header",cols="a,2a"]
 |===
-| Character | Meaning | Example
 
-| `\|`
-| 
-Used to indicate alternative parts of a command (i.e. `or`).
-Needs to be part of a grouping.
-| If the syntax needs to specify either a name or `+*+`, this can be indicated with `+* \| name+`.
+| Content | New location in Operations Manual 
 
-| `+{+` and `+}+`
-| Used to group parts of the command. Commonly found together with `\|`.
-| In order to use the `or` in the syntax summary, it needs to be in a group: `+{* \| name}+`.
+| Database management | link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/[Database administration]
 
-| `[` and `]`
-| Used to indicate an optional part of the command.
-It also groups alternatives together, when there can be either of the alternatives or nothing.
-| If a keyword in the syntax can either be in singular or plural, we can indicate that the `S` is optional with `GRAPH[S]`.
+| Alias management | link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/aliases/manage-aliases-standard-databases/[Database administration -> Managing aliases]
 
-| `+...+`
-|
-Repeated pattern.
-Related to the command part immediately before this is repeated.
-| A comma separated list of names would be `+name[, ...]+`.
+| Server management | link:{neo4j-docs-base-uri}/operations-manual/{page-version}/clustering/[Clustering]
```

Comment:
As above, rather than saying _Clustering -> Managing servers in a cluster_ I think it's better and more conventional to just use the title of the sub-chapter.